R/ocjr-drug-court-SQ780.R

```diff
@@ -1,17 +1,522 @@
+        manufacture | commercial | school_park_church | conspiracy | maintaining |
+        minor | inmate | vehicle | proceeds | impersonate_officer | tax_stamp |
+        a_b_upon,
+      TRUE, FALSE
+    ),
+    # Any drug regex? Not worried about uccs codes with this variable yet
+    regex_drugs = if_else(poss & drug, TRUE, FALSE),
+    # Any property crime regex? Not worried about uccs codes with this variable
+    regex_props = if_else(larceny | receive_stolen | embezzle | misc | pawn | hospitality | forgery, TRUE, FALSE)
+  ) |>
+  arrange(count_as_disposed)
+
+# Andrew note: Gonna try to remove repeated code and make this easier to read
+# Basically just replacing the !(dist | intent | paraphernalia...) bit with the `disqualifying_regex` var above
+# 406777 rows
+# clean_df <- data_clean |>
+#   filter(
+#     (uccs_drug_plus & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (poss & drug & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (uccs_property & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (larceny & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (receive_stolen & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (embezzle & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (misc & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (pawn & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (hospitality & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))|
+#     (forgery & !(dist | intent | paraphernalia | weapon | trafficking | endeavor | wildlife | manufacture | commercial | school_park_church | conspiracy | maintaining | minor | inmate | vehicle | proceeds))
+#   ) |>
```

Comment:
This part with the repeated code (the `!(dist | intent | paraphernalia...` bit) is what I'm trying to avoid; it's just really hard to read and makes it cumbersome to make changes.

Reply:
It's definitely awful and appreciate seeing your rewrite. Makes a lot of sense